CRM/Mailing/Form/Unsubscribe.php

```diff
@@ -119,7 +121,7 @@ public function postProcess() {
     // Email address verified
     $groups = CRM_Mailing_Event_BAO_MailingEventUnsubscribe::unsub_from_mailing($this->_job_id, $this->_queue_id, $this->_hash);
 
-    if (count($groups)) {
+    if (!empty($groups)) {
```

Comment:
This is fine, but FYI for a variable that has definitely been defined (as this one was 1 line above) you don't need count or empty, the if() will cast it to boolean just fine.